elastic-utils-lib/src/cmr/elastic_utils/config.clj

```diff
@@ -19,15 +27,25 @@
   {:default 200000
    :type Long})
 
+(declare gran-elastic-host)
+(defconfig gran-elastic-host
+           "Elastic host or VIP for granule ES cluster."
```

Comment:
The spacing is off here, there older code uses 2 spaces, I am guessing your IDE is putting the lines after the macro name.  We should be consistent.